FitVid/src/Components/SideBar/SideBar.js

```diff
@@ -33,11 +34,16 @@ const SideBar = () => {
           Like
         </NavLink>
 
-        <NavLink to="/playlist" style={findActive} className="sidebar-item">
+        <NavLink to="/playlists" style={findActive} className="sidebar-item">
```

Comment:
Appreciation : Good that you have used active Link using NavLink

Reply:
Thanks Manoj😊